src/Physics/GenericHydro.cc

```diff
@@ -168,7 +168,7 @@ dt(const DataBase<Dimension>& dataBase,
   if (GeometryRegistrar::coords() == CoordinateType::Spherical) {
     Fdiv = +[](const Tensor& DvDxi, const Vector& posi, const Vector& veli) { return DvDxi[0] + 2.0*veli[0]*safeInv(posi[0]); };
   } else if (GeometryRegistrar::coords() == CoordinateType::RZ) {
-    Fdiv = +[](const Tensor& DvDxi, const Vector& posi, const Vector& veli) { return DvDxi.Trace() + veli[0]*safeInv(posi[0]); };
```

Comment:
What was the reason for this change?

Reply:
I reverted this to what it was before, removing the veli[0]*safeInv(posi[0]) term. The extra term crashes the Spheral time step constraint in our 2D RZ problems, where it was stable previously.